configure.ac

```diff
@@ -140,6 +140,7 @@ AC_CHECK_HEADERS([ \
 	sys/mount.h \
 	sys/ndir.h \
 	sys/pstat.h \
+	sys/stat.h \
```

Comment:
do we really need to check for sys/stat.h ?

it is posix, exists even on windows and i've never seen a Linux that doesn't have it

Reply:
I think autoconf checks for it automatically so I can remove this.